build/files.js

```diff
@@ -231,6 +231,22 @@ function CorkedRequest(state) {
       /Buffer\.prototype\.copy\.call\(src, target, offset\);/
     , 'src.copy(target, offset);'
   ]
+  , maybeDefineProperty = [
+        headRegexp
+      , '$1\n' + `
+/* <replacement> */
+var maybeDefineProperty = Object.defineProperty;
+// IE < 9
+if (!maybeDefineProperty || global.document && global.document.documentMode < 9) {
```

Comment:
the problem is that it is not working as expected. The error you are seeing is due to IE 8 broken `Object.defineProperty`. Something in that bit of logic is wrong. I would not like to use the try-catch approach.